File: bit.js
const { Web3, getHtml } = require("blockcloud");
const express = require("express");

const web3 = new Web3(host="127.0.0.1",port=5000);
const app = new express();
console.log(web3.wtc.block.count)

app.get((req,res)=>{
  let user = req.query.username;
  if (!user || user==null) {
    res.send("Cannot find username");
  }
  let contract = web3.wtc.web.contract({username:user,reviewer:"wbank",amount:"100"})
  res.send(getHtml(contract,src="bc/contrract-sdk"))
});

app.listen((5000,'127.0.0.1'), ()=> {
  console.log("Listen on port 5000");
});

File: bit.py
from blockcloud import Web3
from blockcloud.provider import mainNet
from j import complie

web3 = Web3(url="https://mainnet.bangjjn.net:5009")

jCode = """
  import com.net.wcoins.Contract;
  public static void start() {
    //add-string-array
    String <> data = InForm.get("msgData");
    Contract contract = new Contract(data);
    contract.showScreen();
  }
"""

contract = web3.wtc.complie(complie(jCode),src="bc/J-lang",parms={"msgData":{"username":"Ben","reviewer":"wtech","amount":"100"}})